src/scripts/clientUtils.js

```diff
@@ -29,20 +29,43 @@ export async function serverAddress() {
     // if (current) return Promise.resolve(current);
 
     const urls = [];
+    if (window.location.href.indexOf('/web/') !== -1) { // Do we polyfill String.prototype.includes?
+        const split = window.location.href.split('/web/');
+
+        for (let i = split.length - 1; i ; i--) {
```

Comment:
```suggestion
        for (let i = split.length - 1; i >= 0; i--) {
```
(maybe `> 0` instead of `>= 0`)

Reply:
Is it poor practice to use 0 as boolean false like this in JavaScript?
(should be `> 0` if so)

Reply:
I think it makes the code harder to read (but I mostly use type-safe languages so it's probably because of that)